eif_utils/src/lib.rs

```diff
@@ -26,11 +28,17 @@ use std::collections::BTreeMap;
 use std::ffi::CString;
 use std::fmt::Debug;
 use std::fs::File;
-use std::io::{Read, Seek, SeekFrom, Write};
+use std::io::{BufRead, BufReader, Read, Seek, SeekFrom, Write};
 use std::mem::size_of;
 use std::path::Path;
+use std::time::SystemTime;
 
 pub const DEFAULT_SECTIONS_COUNT: u16 = 2;
+pub const NITROCLI_SPECS_PATH: &str = "../../SPECS/aws-nitro-enclaves-cli.spec";
+#[cfg(target_arch = "x86_64")]
+pub const KERNEL_CONFIG_PATH: &str = "../../blobs/x86_64/bzImage.config";
+#[cfg(target_arch = "aarch64")]
+pub const KERNEL_CONFIG_PATH: &str = "../../blobs/aarch64/bzImage.config";
```

Comment:
for aarch64 the config file is not called bzImage.config anymore

Reply:
Image.config?

Reply:
Yes, that's the new name.